cmd/telemetrygen/internal/common/config.go

```diff
@@ -36,6 +37,12 @@ func (v *KeyValue) Set(s string) error {
 		return errFormatOTLPAttributes
 	}
 	val := kv[1]
+	parsedBool, err := strconv.ParseBool(val)
```

Comment:
This accepts also 0 and 1, which is ambiguous (I would expect if I set an attribute to `1` it would be set to the integer 1, not the boolean value true). Could we just parse `true` and `false`? Or explicitly exclude `0` and `1`